lookout/style/typos/generation.py

```diff
@@ -375,18 +383,26 @@ class DummyModel(Model):
         for key, val in self.wv.vocab.items():
             vocab_strings[val.index] = key
             vocab_counts[val.index] = val.count
-        hash2index = numpy.zeros(len(self.wv.hash2index), dtype=numpy.uint32)
-        for key, val in self.wv.hash2index.items():
-            hash2index[val] = key
+        if isinstance(self.wv.buckets_word, dict):
```

Comment:
How come we have an "if" here?

Reply:
It sometimes can be a dict, and sometimes a NoneType.

When I wrote this code a couple of months ago with `gensim==3.7.2`, there was a dict in the model, loaded with `load_facebook_vectors`. RIght now there's `NoneType`, but probably it can become `dict` when smth will be cached, I'm not sure.

Reply:
Ok, this attribute seem to not be used when the model is loaded from the facebook dump, so I removed it.